lib/DDGC/Web/App/Base.pm

```diff
@@ -0,0 +1,33 @@
+package DDGC::Web::App::Base;
+
+use Import::Into;
+
+use strict;
+use warnings;
+use utf8;
+
+use Dancer2;
+use Dancer2::Plugin::DBIC;
+use DDGC::Web::Plugin::Config;
+use DDGC::Web::Plugin::Session;
+
+{   no warnings 'redefine';
+    sub import {
+        my ($caller, $filename) = caller;
+        for (
+          qw/
+            strict
+            warnings
+            utf8
+            Dancer2
+            Dancer2::Plugin::DBIC
+            DDGC::Web::Plugin::Config
+            DDGC::Web::Plugin::Session
+          /
+        ) {
+            $_->import::into($caller);
+        }
+    }
+};
+
+1;
```

Comment:
Re `package DDGC::Web::App::Base`

`Import::Into` performs an import step on any module of your choosing (so `'Foo'->import::into('Bar')` effectively inserts `use Foo` into `package Bar`).

This makes `DDGC::Web::App::Base` a "base module" (**not** a base class) which does the setup, exports and everything else `use`ing those modules would.

The reason for this is, each app or service has a number of required modules to do config and setup. While you can copypaste from a working application, this is prone to error. I wanted to make creating a new app much simpler. ([See this diff](https://github.com/duckduckgo/community-platform/commit/d9a547f))

@MariagraziaAlastra @nilnilnil @jdorweiler @russellholt Any thoughts? Neat or horrific? I'm not sure.